api_tests/chronos/views/test_chronos_submission_list.py

```diff
@@ -215,7 +219,10 @@ def submission_fresh(self, preprint, journal_two, submitter):
 
     @pytest.fixture()
     def url(self, preprint):
-        return f'/_/chronos/{preprint._id}/submissions/'
+        preprint_id = preprint._id
+        if VersionedGuidMixin.GUID_VERSION_DELIMITER in preprint._id:
+            preprint_id = preprint._id.split(VersionedGuidMixin.GUID_VERSION_DELIMITER)[0]
+        return f'/_/chronos/{preprint_id}/submissions/'
```

Comment:
Same issue, instead of modifying the test to use base guid, we should update the api code to support versioned guid.